File: programers_Lv1/같은 숫자는 싫어.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12906?language=python3
def solution(arr):
    temp = -1
    answer = []
    for n in arr:
        if temp != n:
            answer.append(n)
            temp =n
    return answer


print(solution([1,1,3,3,0,1,1]))

File: programers_Lv1/같은 숫자는 싫어.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12906?language=javascript
function solution(arr) {
  let temp = -1;
  const answer = [];
  arr.forEach((el) => {
    if (el !== temp) {
      temp = el;
      answer.push(el);
    }
  });

  return answer;
}

console.log(solution([1, 1, 3, 3, 0, 1, 1]));
